Kernel/TTY/TTY.h

```diff
@@ -41,10 +41,10 @@ class TTY : public CharacterDevice {
     }
 
     KResult set_termios(const termios&);
-    bool should_generate_signals() const { return m_termios.c_lflag & ISIG; }
-    bool should_flush_on_signal() const { return !(m_termios.c_lflag & NOFLSH); }
-    bool should_echo_input() const { return m_termios.c_lflag & ECHO; }
-    bool in_canonical_mode() const { return m_termios.c_lflag & ICANON; }
+    bool should_generate_signals() const { return ISIG == (m_termios.c_lflag & ISIG); }
+    bool should_flush_on_signal() const { return 0 == (m_termios.c_lflag & NOFLSH); }
+    bool should_echo_input() const { return ECHO == (m_termios.c_lflag & ECHO); }
+    bool in_canonical_mode() const { return ICANON == (m_termios.c_lflag & ICANON); }
```

Comment:
More Yoda conditions (To reiterate: I like them, but so far Serenity doesn't typically use them, so let's try to avoid them to keep the code uniform and non-surprising.)